c7n/resources/ebs.py

```diff
@@ -210,6 +211,45 @@ def get_instance_mapping(self, resources):
         return {i['InstanceId']: i for i in instances}
 
 
+@filters.register('kms-alias')
+class KmsKeyAlias(ValueFilter):
+
+    schema = type_schema('kms-alias', rinherit=ValueFilter.schema)
+
+    def get_kms_aliases(self):
+        from c7n.resources.kms import KeyAlias
+        manager = KeyAlias(self.manager.ctx, {})
+        return [k for k in manager.resources()]
+
+    def process(self, resources, event=None):
+
+        key_aliases = self.get_kms_aliases()
```

Comment:
given top level import the extra method seems a little superflous, ie vs just key_aliases = KeyAlias(self.manager.ctx, {}).resources()